doc/stages/readers.ept.rst

```diff
@@ -16,16 +16,14 @@ This example downloads a small area around the the Statue of Liberty from the Ne
 
 .. code-block:: json
 
-    {
-      "pipeline": [
```

Comment:
Did something change that PDAL is now able to parse the pipeline array directly? I think you've told me that Entwine has been able to do this for a while. Just wonder who the audience is here.

Reply:
I think this went in for 1.8, but yes, you can just provide an array of stages now.

Reply:
It was #2183.  It missed 1.8 I think but has been on master since shortly afterward.

Reply:
That sounds right.